invenio.cfg

```diff
@@ -550,3 +550,14 @@ VOCABULARIES_NAMES_SCHEMES = {
     "lcds": {"label": _("CDS"), "validator": schemes.is_legacy_cds, "datacite": "CDS"},
 }
 """Names allowed identifier schemes."""
+
+
+# Invenio Stats
+# =============
+
+# We need to add explicitely the `filter_robots` as the migrated legacy statistics are
+# were not being processed through the event preprocessors
```

Comment:
Improve comment? are/were? and where did we add the `filter_robots`?

Reply:
leftover I will remove it